internal/pkg/cli/app_delete.go

```diff
@@ -356,12 +375,19 @@ func (o *deleteAppOpts) deleteAppConfigs() error {
 }
 
 func (o *deleteAppOpts) deleteWs() error {
-	o.spinner.Start(fmt.Sprintf(fmtDeleteAppWsStartMsg, workspace.SummaryFileName))
-	if err := o.ws.DeleteWorkspaceFile(); err != nil {
-		o.spinner.Stop(log.Serrorf("Error deleting %s file.\n", workspace.SummaryFileName))
-		return fmt.Errorf("delete %s file: %w", workspace.SummaryFileName, err)
+	ws, err := o.existingWorkSpace()
+	if err == nil {
+		// When there's a local application summary.
+		summary, err := ws.Summary()
+		if err == nil && summary.Application == o.name {
+			o.spinner.Start(fmt.Sprintf(fmtDeleteAppWsStartMsg, workspace.SummaryFileName))
+			if err := ws.DeleteWorkspaceFile(); err != nil {
+				o.spinner.Stop(log.Serrorf("Error deleting %s file.\n", workspace.SummaryFileName))
+				return fmt.Errorf("delete %s file: %w", workspace.SummaryFileName, err)
+			}
+			o.spinner.Stop(log.Ssuccessf(fmt.Sprintf(fmtDeleteAppWsStopMsg, workspace.SummaryFileName)))
+		}
```

Comment:
is it possible to reverse the `if` condition and reduce nestings?

Reply:
done. reversed the condition of if statements